src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const Btnrefs = {
```

Comment:
```suggestion
const buttonRefs = {
```
Use camelCase. Also don't shorten names, especially if it's about 2-3 chars

Reply:
done